shared/output/cloud/compute_small/lib/google/cloud/compute/v1/addresses/rest/client.rb

```diff
@@ -310,10 +329,10 @@ def delete request, options = nil
               #   @param region [::String]
               #     Name of the region for this request.
               # @yield [result, response] Access the result along with the Faraday response object
-              # @yieldparam result [::Google::Cloud::Compute::V1::Address]
+              # @yieldparam result [::Gapic::GenericLRO::Operation]
               # @yieldparam response [::Faraday::Response]
               #
-              # @return [::Google::Cloud::Compute::V1::Address]
+              # @return [::Gapic::GenericLRO::Operation]
```

Comment:
How did this change happen? Is this going to cause a breaking change in the actual compute library?

Reply:
Excellent catch, thank you!
This one was triggered by changes in the proto which were a copy-paste mistake I committed in a different PR. It appeared after a rebase and I didn't notice it when reviewing.
The mistake assigned a nonstandard LRO service to this method, even though it was not returning an `Operation` that said LRO service could poll.

- I have removed the copy-paste mistake
- And also I have added a check that will throw an error if a method gets assigned a NonstandardLRO service that cannot poll its return type.